### PR TITLE
Rpamidimarri 364292

### PR DESCRIPTION
**Pull request by rpamidimarri-okta**
## Description:
- We are going to deprecate the usage of device fingerprints for the new device behavior detection. We should doc this in the API dev docs.
- This is not related to any monolith change

### Resolves:

* [OKTA-364292](https://oktainc.atlassian.net/browse/OKTA-364292)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 1]
```diff
@@ -881,6 +881,7 @@ Include the `X-Device-Fingerprint` header to supply a device fingerprint. The `X
...
 * If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. See [New Device Behavior Detection](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_proc_security_behavior_detection).
+> **Note:** The use of the `X-Device-Fingerprint` header for new device security behavior detection is deprecated. Starting April 12 2021, we are going to enable [improvements to the new device security behavior](https://help.okta.com/en/prod/Content/Topics/Security/improved-new-device-behavior-detection.htm) for all the existing tenants. Once the improvements are rolled out, new device security behavior would only rely on the `deviceToken` in the [Context Object](#context-object) and would not rely on the `X-Device-Fingerprint` header.
```
Comment (rpamidimarri-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:884]: We would have to discuss with PM on the date we want to include in this doc. I chose April 12th as a place holder for now.

Comment (dannavarro-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authn/index.md:884]: I am skeptical of including a date at all. We have already communicated a number of deadlines and we have missed all of them so far

**Comment by dannavarro-okta** [comment]
can you attach a screenshot of the change deployed locally?

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 2]
```diff
@@ -881,6 +881,7 @@ Include the `X-Device-Fingerprint` header to supply a device fingerprint. The `X
...
 * If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. See [New Device Behavior Detection](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_proc_security_behavior_detection).
+> **Note:** The use of the `X-Device-Fingerprint` header for new device security behavior detection is deprecated. Starting April 12 2021, we are going to enable [improvements to the new device security behavior](https://help.okta.com/en/prod/Content/Topics/Security/improved-new-device-behavior-detection.htm) for all the existing tenants. Once the improvements are rolled out, new device security behavior would only rely on the `deviceToken` in the [Context Object](#context-object) and would not rely on the `X-Device-Fingerprint` header. The new or unknown device email notification feature would continue to rely on the `X-Device-Fingerprint` header.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:884]:
```suggestion
> **Note:** The use of the `X-Device-Fingerprint` header for new device security behavior detection is deprecated. Starting April 12 2021, we are going to enable [improvements to the new device security behavior](https://help.okta.com/en/prod/Content/Topics/Security/improved-new-device-behavior-detection.htm) for all the existing tenants. Once the improvements are rolled out, new device security behavior will only rely on the `deviceToken` in the [Context Object](#context-object) and will not rely on the `X-Device-Fingerprint` header. The new or unknown device email notification feature will continue to rely on the `X-Device-Fingerprint` header.
```

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**